custom/languages/en_UK.json

```diff
@@ -924,6 +928,8 @@
     "installer/reload_page": "Reload page",
     "installer/requirements": "Requirements:",
     "installer/requirements_error": "You must have all of the required extensions installed, and have correct permissions set, in order to proceed with installation.",
+    "installer/select_modules": "Select Modules",
+    "installer/select_modules_details": "Select the modules you would like to enable on your site. These toggled later on through the admin panel.",
```

Comment:
Should read "These **can** be toggled later on through the admin panel."

Reply:
Thanks!